docs/cloud-tutorials/best-practices.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 8
+title: Best practices
+description: Tips and tricks for succesfully using DBOS Cloud
+---
+
+This section dives into specific behaviors and corner cases you might encounter when using DBOS Cloud.
+
+## Managing applications
+
+### Registration
```

Comment:
Since we're merging registration with deploy from the CLI, we can fold this into the deployment section.